src/test/java/formflow/library/email/MailgunEmailClientTest.java

```diff
@@ -116,17 +118,17 @@ public void mailgunWillSendWithMinimumInfoAndAttachments() throws Exception {
   }
 
   @Test
-  public void mailgunWillSendAllProvidedInfo() throws Exception {
+  public void mailgunWillSendAllProvidedInfo() {
     final ArgumentCaptor<Message> captor = ArgumentCaptor.forClass(Message.class);
-    doReturn(null).when(mailgunMessagesApi).sendMessage(any(), captor.capture());
+    doReturn(messageResponse).when(mailgunMessagesApi).sendMessage(any(), captor.capture());
 
     String expectedSubject = "sendEmail with all parameters";
     String expectedRecipient = "test@example.com";
     List<String> expectedEmailsToCC = Arrays.asList("one@test.com", "two@test.com", "three@test.com");
     List<String> expectedEmailsToBCC = Arrays.asList("secret-one@test.com", "secret-two@test.com");
     String expectedBody = "Email body";
     List<File> expectedAttachments = Arrays.asList(new File("src/test/resources/pdfs/blankPdf.pdf"));
-    Boolean expectedRequireTls = true;
+    Boolean expectedRequireTls = Boolean.TRUE;
```

Comment:
What's the difference between `true` and `Boolean.TRUE`?

Reply:
Boolean object corresponding to true, while true is the primitive boolean value of true.  It changes nothing but it removes a complaint intellij makes